src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/BasePageObject.java

```diff
@@ -187,8 +187,8 @@ protected void scrollAndClick(WebElement element, int offset) {
 
   protected void scrollToElement(WebElement element) {
     try {
-      ((JavascriptExecutor) driver).executeScript("var x = $(arguments[0]);"
-          + "window.scroll(0,parseInt(x.offset().top - 100));", element);
+      ((JavascriptExecutor) driver).executeScript(
+          "var x = $(arguments[0]); window.scroll(0,parseInt(x.offset().top - 100));", element);
     } catch (WebDriverException e) {
       if (e.getMessage().contains(XSSContent.NO_JQUERY_ERROR)) {
         PageObjectLogging.log("JSError", "JQuery is not defined", false);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "JSError" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "JQuery is not defined" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)